rules/summary-recording-sprint-reviews/rule.md

```diff
@@ -0,0 +1,71 @@
+---
+type: rule
+title: Do you create a summary and recording at the end of Sprint Reviews for
+  important stakeholders?
+uri: summary-recording-sprint-reviews
+authors:
+  - title: Jack Pettit
+    url: https://ssw.com.au/people/jack-pettit/
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan/
+related:
+  - tech-check
+  - record-a-quick-and-dirty-done-video
+  - rules-to-better-video-recording
+created: 2024-04-15T00:46:35.508Z
+guid: 99674806-7353-4d39-ac9a-219620d4e107
+---
+Creating a comprehensive summary and recording of Sprint Reviews is vital for effective communication and documentation, especially when key stakeholders are unable to attend these meetings. Imagine finishing a Sprint Review without a clear record—stakeholders are left without insights into project progress, leading to potential misalignments and missed opportunities for feedback.
```

Comment:
I'm not sure I agree with this.  Sprint Reviews ARE the place for stakeholders to attend, be kept in the loop, ask questions and feed changes into the backlog.  I don't think a video is mandatory to achieve this.

Reply:
I've updated it to address this issue. If not happy, submit another PR.